### DISCUSSION TITLE
但theme组件报null的问题

### DISCUSSION THREAD

**Discussion by ACSE-cr22** [ant-design/ant-design#48453] [Q&A]
有没有人复制布局的代码，然后theme组件报空的问题
<img width="469" alt="8cc2eb35c7da7244f5645cd71123c16" src="https://github.com/ant-design/ant-design/assets/110286180/83d702cb-528f-410a-92d6-451671bd0f42">
<img width="471" alt="b36249bceba4f7f8d1d8476af8c60c7" src="https://github.com/ant-design/ant-design/assets/110286180/425340d7-c7db-4aa2-acd6-bea633c6f0cf">
<img width="746" alt="3367b38b0664876c2464c90e0a99281" src="https://github.com/ant-design/ant-design/assets/110286180/6c5fdfdc-ab3d-424a-ab92-ffb91bc8a4e8">

**Comment by MadCcc** [comment]
你的 antd 版本是？
